.github/workflows/android-build.yml

```diff
@@ -35,6 +33,6 @@ jobs:
           mkdir "${HOME}/vulkan-sdk"
           tar -xf vulkansdk-linux-x86_64-1.3.216.0.tar.gz -C "${HOME}/vulkan-sdk"
           echo "VULKAN_SDK=${HOME}/vulkan-sdk/1.3.216.0/x86_64" >> $GITHUB_ENV
-      - name: Build the sample for Android
+      - name: Build all projects
         run: |
-          ./gradlew ${{ matrix.task }}
+          ./gradlew buildRelease
```

Comment:
What is the required storage for those? Are we close to the limit or not?

Reply:
This seems to add about 9 GB, and adds to the <2 GB of the repository itself. The runners are guaranteed to have at least 14 GB, so its not close to the limit. FWIW, in my tests the available space was actually 24 GB on the runners, but I think only 14 is actually guaranteed.

Reply:
All right, so we do have some margin left 😊
Let's move back to 1 workflow for this then.